.github/workflows/dapr-test.yml

```diff
@@ -55,8 +55,13 @@ env:
   AZURE_ARM_REGIONS: "eastus"
   # Container registry where to cache e2e test images
   DAPR_CACHE_REGISTRY: "dapre2eacr.azurecr.io"
+  # Name of the Azure Key Vault resource used in tests
+  # The credentials defined in AZURE_CREDENTIALS must have permissions to perform operations in this vault
+  AZURE_KEY_VAULT_NAME: "dapre2ekv"
```

Comment:
For Azure, I have pre-provisioned a Key Vault in our test subscription, which is accessible using the same Azure AD application we use to deploy resources. Deploying a new KV on each test is complex due to the need to also generate keys and manage permissions via Azure AD, so there's a single one that all tests on AKS use.

This should not be a problem, since capacity isn't a concern even if we run multiple tests in parallel. Additionally, Key Vault's pricing is essentially just per API call, so keeping a Key Vault pre-provisioned incurs only marginal costs.